apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -915,6 +915,25 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
       }
 
       blocklyWrapper.KeyboardNavigation = new KeyboardNavigation(workspace);
+
+      // Now that the shortcutModal is initialized, we can add a keydown
+      // event listener to the modal to close it when the Escape key is pressed.
+      const shortcutModal = document.querySelector('.shortcut-modal');
```

Comment:
If we load multiple levels, `blocklyWrapper.KeyboardNavigation` is disposed and re-instantiated. Do these steps remove and re-create the `.shortcut-modal` element? I'm wondering whether this listener needs to be added each time we change levels, as you have it here, or only once per page load.